templates/blog/user_list.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+{% block title %}
+
+  <title>Users</title>
+{% endblock %}
+{% block content %}
+  <h1>Users</h1>
+  <p>{{ users_list }}</p>
```

Comment:
The variable `users_list` is used here, but it should be `user_list` to match the context provided in the template logic. Consider removing or correcting this line if it's not needed.